external/sketch/ansible/roles/proxypass-setup/README.md

```diff
@@ -0,0 +1,54 @@
+proxypass-setup
+=========
+
+Optional role.
+
+This will automatically setup the Proxy Protocol chain from all edge nodes to middle. And middle to proxy
+
+Best used for when using only one middle sketch.
+
+This assumes you are pointing middle to proxy01.
+
+Requirements
+------------
+
+Must be ran after initial Sketch setup as nginx is required to be installed and loaded. Make sure you copy below into your playbook
+
+If using OCI replace `REPLACE ME` with the following:
+
+"jq '.resources[] | select(.type == \"oci_core_instance\" and .name == \"proxy\") | .instances[0].attributes.public_ip' ../../oci/terraform.tfstate"
```

Comment:
This should be done with terraform output. `external/PROVIDER/output.tf`

Something like this for OCI
```
output "proxy_public_ips" {
  value = [for ip in oci_core_instance.proxy : ip.public_ip]
}
```

And something like this for AWS:
```
output "proxy_public_ips" {
  value = [for ip in aws_instance.proxy : ip.public_ip]
}
```

This will just print out the IP addresses for the proxies. If you wanted to just print the ip for `proxy01` then do something like this:

```
output "proxy01_public_ip" {
  value = oci_core_instance.proxy[0].public_ip
}
```